From: Arnaud Ferraris <aferraris@debian.org>
Date: Tue, 8 Oct 2024 10:03:26 +0200
Subject: [PATCH] arm64: dts: sdm670-google-common: set chassis-type property

The Pixel 3a/3a XL are mobile phones, we should ensure userspace can be
aware of that by setting the `chassis-type` property.

Signed-off-by: Arnaud Ferraris <aferraris@debian.org>
---
 arch/arm64/boot/dts/qcom/sdm670-google-common.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/sdm670-google-common.dtsi b/arch/arm64/boot/dts/qcom/sdm670-google-common.dtsi
index d56665f..becb3fe 100644
--- a/arch/arm64/boot/dts/qcom/sdm670-google-common.dtsi
+++ b/arch/arm64/boot/dts/qcom/sdm670-google-common.dtsi
@@ -32,6 +32,8 @@
 /delete-node/ &rmtfs_mem;
 
 / {
+	chassis-type = "handset";
+
 	aliases {
 		hsuart0 = &uart6;
 	};
